changelog.md

```diff
@@ -1,3 +1,5 @@
+- New: adjustTables allows for setting up tables for expand / collapse behavior without initially collapsing them
```

Comment:
Will want to add `v5.0.0` on the line above since this change isn't backward compatible.

Reply:
It is backwards compatible, `collapseTables` is still available

Reply:
Oops my bad!